### DISCUSSION TITLE
Expose Network protocols

### DISCUSSION THREAD

**Discussion by volga629-1** [nvim-tree/nvim-tree.lua#2589] [Ideas]
Hello Everyone,
This just general discussion/ request  to enable Nread  or e/edit  use of network protocols like SSH , SFTP
Right now I am use 

```
	vim.g.loaded_netrw = 1
       vim.g.loaded_netrwPlugin = 1
```

This project is  exactly what is need it.

```
https://github.com/zenbro/mirror.vim#installation

```

**Comment by alex-courtis** [comment]
netrw can be enabled however you may experience some issues with netrw opening instead of nvim-tree

See `:help nvim-tree-netrw` for options that attempt to allow netrw and nvim-tree to play nicely.
